addons/mrp/tests/test_bom.py

```diff
@@ -2235,3 +2235,11 @@ def test_mrp_bom_product_catalog(self):
 
         self.start_tour(url, 'test_mrp_bom_product_catalog', login='admin')
         self.assertEqual(len(bom.bom_line_ids), 1)
+
+    def test_manufacture_from_bom(self):
```

Comment:
Isn't it possible to include it in another test. We should try to limit the number of tour when possible since it takes a lot of times and ressources to launch

Reply:
Unfortunately, I haven't found any other tests that would look like they enter the "BoM Overview" screen. That's why I created a new one. But perhaps I've missed something, so if you know of a more suitable place, I will move my code there.